### DISCUSSION TITLE
Nothing is right in lovelace

### DISCUSSION THREAD

**Discussion by Adra-Yn** [home-assistant/frontend#12192] [Dashboard] [answered]
Hello,

Have you already seen this before ? 
![image](https://user-images.githubusercontent.com/61202466/161271477-b8baabf2-9b52-4ecc-bca1-e5a27df5571d.png)
I make a zoom for more clarity. 
I've had no change on my lovelace config. 
And I don't know how fix that.

Someone have this problem too ? 

Thx.

**Comment by frenck** [comment, marked as answer]
This is not a Home Assistant issue, but an April fools joke of a custom card.

https://github.com/thomasloven/lovelace-card-mod/issues/192

**Reply by Adra-Yn** [reply]
Haha, :') Ok thx for rapid answer :)
